Add options for dynmap customization

tinyoverflow opened the pull request:
This PR contains a commit which adds the following options for the dynmap integration:
* marker-set-label
* missing-name-placeholder

The `marker-set-label` is the string which is used in the dynmap layer list to identity the layer itself. In addition to that, currently the placeholder string, when a claim doesn't have a display name, is hardcoded to "none". To change this, the option `missing-name-placeholder` can be used.

In addition to that, it adds the following placeholders to the dynmap popup window:
- `%width%`
- `%length%`
- `%height%`
- `%area%`
- `%volume%`

It also removes the MOD_ID constant inside the DynmapProvider, as it isn't used anymore.

Furthermore it refactors the DynmapProvider by reducing duplicate code inside the class and also by making the code itself more readable.

_Please note: I do not have access to all the plugins that are referenced in this plugin. Therefore I cannot compile or test anything myself. Please point it out to me when something is broken or if I've missed something._

tinyoverflow committed:
Make dynmap markerset label and claim name fallback customizable

bloodmc commented on bukkit/src/main/java/com/griefdefender/hooks/provider/DynmapProvider.java:
```diff
@@ -100,11 +99,17 @@ private String getWindowInfo(Claim claim, AreaMarker marker) {
+        info = info.replace("%width%", Integer.toString(claim.getWidth()));
+        info = info.replace("%length%", Integer.toString(claim.getLength()));
+        info = info.replace("%height%", Integer.toString(claim.getHeight()));
+        info = info.replace("%area%", Integer.toString(claim.getArea()));
```
This is a duplicate area check. Remove the other. Also update your PR notes to mention these changes.

tinyoverflow replied:
Duplicate placeholder has been removed.

tinyoverflow committed:
Moved HTML wrapper to return statement to prevent accidental modification

tinyoverflow committed:
Extract redundant player-list-generator code into its own method and update the way the claim name gets fetched in the DynmapProvider

tinyoverflow commented:
I've removed the duplicate placeholder (didn't saw that, sorry), updated my PR description and also added two more commits which deduplicate code and also move the HTML container wrapper to the end of the `getWindowInfo` method.
